TODO.md

```diff
@@ -7,6 +7,18 @@ Scratchpad for random ideas of things that need done.
 * Verify RDP enable
 * Linux repos
 * Whole buncha stuff for profile::cfcc::camper
+Charlie's $0.02
+
+* jdk8 > jdk10
+* remove eclipse
+* add a good text editor (I use notepad2-mod; I think lots of the others are overkill and overwhelming, nbd though)
+* Add registry entry to allow edit of any file from context menu in classesroot/*/editinnotepad[Edit in Notepad]/command/notepad.exe %1
+* add python 2.7? I recall we use odd python stuff that has compatibility issues and thought that might be one; I'm okay leaving it out if it's not an issue
+* We still do alice? is it update?
+* Any way to pre-configure SDKs in PyCharm and IDEA?
+* Add games? Especially one with a hacked up q3 config
```

Comment:
How so?

Reply:
IDK if there are packages or we can just dump a zip or something

Reply:
I already install Q3 and Minecraft. Steam games are difficult because you can't really automate their installation, and doing huge file transfers via Puppet generally leads to bad times. Hence for that why I've been digging into the local cache solution more as that at least cuts down the install time to a few minutes.